src/services/themeService.tsx

```diff
@@ -28,7 +29,8 @@ const getGitHubThemeData = async (apiTheme: ApiTheme) => {
 	const cssStylesUrl = `${contentUrl}/${cssStylesFile}`;
 
 	// todo: fetch from github url (can use jsdelivr cache too)
-	const authorImg = '';
+  
+	const authorImg = `https://avatars.githubusercontent.com/${meta.author}`
```

Comment:
Instead of `meta.author`, use `meta.github` which represents their github handle. That'll be more consistent since the `author` field is really a generic name and up to how the author wishes to be addressed 😊